docs/chat-configuration/ui-customization/incoming-message/readmore-article.md

```diff
@@ -36,6 +36,21 @@ val customProvider = ChatUIProvider(context).apply {
         articleUIConfig.verticalMargin = x to y // Pair of top, bottom margin values, in pixels
 
         channelsUIProvider = ... // A QuickOptionUIProvider object
+        
+        // Configure the main background of the Article
+        background = ....
+
+         // Configure the article title
+        title.apply {
+            font = ... // A StyleConfig object
+            background = ... // A color int
```

Comment:
1. u can add an example to the values on the comment
2. emphasize that the size value should be in px